tox.ini

```diff
@@ -1,19 +1,31 @@
 [tox]
 minversion=2.3.1
 envlist =
-    py{27,35}-ansible22-{pylint,unit,flake8,yamllint,generate_validation}
+    py{27,35}-ansible22-lint
+    py{27,35}-ansible22-unit
 skipsdist=True
 skip_missing_interpreters=True
 
 [testenv]
 deps =
+    wheel
     -rtest-requirements.txt
-    py35-flake8: flake8-bugbear
-    ansible22: ansible~=2.2
+    lint: -rtest-requirements-lint.txt
+    unit: -rtest-requirements-unit.txt
+    py35-lint: flake8-bugbear
+    ansible22: ansible==2.2.0.0
+
+whitelist_externals =
+    echo
```

Comment:
Hmm I see http://tox.readthedocs.io/en/latest/config.html#confval-whitelist_externals=MULTI-LINE-LIST

Where do we use `echo`?